test/account.test.js

```diff
@@ -116,6 +117,8 @@ describe('with deploy contract', () => {
         const newPublicKey = await nearjs.connection.signer.createKey(contractId, testUtils.networkId);
         const data = [...fs.readFileSync(HELLO_WASM_PATH)];
         await workingAccount.createAndDeployContract(contractId, newPublicKey, data, HELLO_WASM_BALANCE);
+        setLogLevel(LogLevel.INFO);
```

Comment:
Should we do the same in all test files?

Reply:
hmm. I'm not sure what is better. This one needed it because we are checking that some logs are printed.

But right now the tests print too much info, so I sort of like this only here.